superset-frontend/src/dashboard/util/getFilterScopeFromNodesTree.js

```diff
@@ -22,11 +22,29 @@ import { flatMap, isEmpty } from 'lodash';
 import { CHART_TYPE, TAB_TYPE } from './componentTypes';
 import { getChartIdAndColumnFromFilterKey } from './getDashboardFilterKey';
 
+function getChartsFromTabsNotInScope({ tabs = [], tabsInScope = [] }) {
```

Comment:
maybe `getChartIdsFromTabsNotInScope`? To clarify that this isn't returning chart objects

Reply:
fixed.